app/js/modal.ts

```diff
@@ -56,6 +56,7 @@ function createEditModal(userInfo: BaseUser, tests: Array<Test>) {
     })
     .then(() => {
         addEditModalSubmitEventListener()
+        document.querySelectorAll(".close-edit-user").forEach(button => button.addEventListener('click', closeDialog));
```

Comment:
`querySelectorAll()` gets all the elements that have `.close-edit-user` - in this case only one. then you `forEach` over a list of just 1. you might as well just use `querySelector()` right?

Reply:
We now have 2 elements with the class of .close-edit-user so have retained the querySelectorAll :o)